region/norfair/east.json

```diff
@@ -1470,115 +1709,197 @@
           "to": [
             {
               "id": 1,
-              "requires": [
-                {"or": [
-                  {"canComeInCharged":{
-                    "fromNode": 2,
-                    "framesRemaining": 30,
-                    "shinesparkFrames": 25
-                  }},
-                  {"canComeInCharged":{
-                    "fromNode": 3,
-                    "framesRemaining": 30,
-                    "shinesparkFrames": 35
-                  }},
-                  "canSpringBallWalljumpMidAir",
-                  {"and": [
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "canFly",
+                      {"and": [
+                        "HiJump",
+                        "canSpringBallJumpMidAir"
```

Comment:
requiring springball jump in a base strat 😰

Reply:
I split them off